frontend/packages/kubevirt-plugin/src/components/ssh-service/SSHForm/ssh-form.scss

```diff
@@ -0,0 +1,4 @@
+.SSHForm-main {
+  display: inline-flex;
+  justify-content: space-between;
+}
```

Comment:
missing line ending

Reply:
is it critical? eslint isn't auto fixing/erroring about it and vs code save like this :)